arch/Alpha/AlphaInstPrinter.c

```diff
@@ -0,0 +1,100 @@
+/* Capstone Disassembly Engine */
+/* By Dmitry Sibirtsev <sibirtsevdl@gmail.com>, 2023 */
+
+#ifdef CAPSTONE_HAS_ALPHA
+
+#include <platform.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "../../utils.h"
+#include "../../MCInstPrinter.h"
+
+#include "AlphaLinkage.h"
+#include "AlphaMapping.h"
+
+static const char *getRegisterName(unsigned RegNo);
+
+static void printInstruction(MCInst *, uint64_t, SStream *);
+
+static void printOperand(MCInst *MI, int OpNum, SStream *O);
+
+
+#define GET_INSTRINFO_ENUM
+
+#include "AlphaGenInstrInfo.inc"
+
+#define GET_REGINFO_ENUM
+
+#include "AlphaGenRegisterInfo.inc"
+
+static inline void fill_alpha_register(MCInst *MI, uint32_t reg)
```

Comment:
Move those kind of `detail` filling functions to `AlphaMapping.*` please. So they follow the principle design of all `auto-sync` archs.

Reply:
Also implement the signature just as in PPC or ARM. For example you need to pass the `OpNum`, because in `AlphaGenCSMappingInsnOp.inc` everything is indexed by the LLVM `OpNum`